### DISCUSSION TITLE
Entity interact event firing twice on the same action?

### DISCUSSION THREAD

**Discussion by KianBrose** [Minestom/Minestom#2094] [Help] [answered]
![image](https://github.com/Minestom/Minestom/assets/62078964/17e2ae9a-b24d-433c-9d73-90db972eec25)
for some reason this seems to be triggering twice on every interaction, why?

**Comment by cosrnic** [comment, marked as answer]
The event fires once per hand, you can check which hand using `event.getHand()`.

**Reply by KianBrose** [reply]
This was correct, thank you
